test/topology_random_failures/cluster_events.py

```diff
@@ -67,6 +67,14 @@ def add_deselected_metadata(fn: Callable[P, T]) -> Callable[P, T]:
 #       >>> await anext(cluster_event, None)
 
 
+@deselect_for(
```

Comment:
By the way, I didn't understand what this "delesect_for" does. Why are we using it and not `@xfail` or `@skip` on specific tests?

Reply:
The logic behind this is following: the full matrix of tests is generating by [pytest_generate_tests](https://github.com/scylladb/scylladb/blob/3e22998dc13156b373e24688371216403938abc2/test/topology_random_failures/test_random_failures.py#L47-L66) hook. Then we pick one random test from the matrix. And we want to run something useful. Not a skip or xfail. So, this decorator puts hints for the hook to remove (deselect) tests from the matrix.